core/nodejsAction/Dockerfile

```diff
@@ -2,56 +2,20 @@ FROM buildpack-deps:trusty-curl
 
 ENV DEBIAN_FRONTEND noninteractive
 
-# Initial update and some basics.
-# This odd double update seems necessary to get curl to download without 404 errors.
-RUN curl -sL https://deb.nodesource.com/setup_0.12 | bash - && \
-  apt-get install -y nodejs imagemagick
-
```

Comment:
there is another disadvantage to the `package.json` - it now means to run the service locally all those packages have to be installed as well.

maybe we can use deps and dev-deps instead?